[APPS-4837] Accept updates to OpenAPI schema file through console command (spryker-sdk/sync-api) (#12)
spryker-sdk/sync-api: README.md

The `vendor/bin/syncapi schema:openapi:validate` validates an OpenAPI file.

## Create code from an existing OpenAPI

The `vendor/bin/syncapi code:openapi:generate` reads an existing OpenAPI file and creates code out of it.

## Update an OpenApi file with provided OpenApi schema

> The `vendor/bin/syncapi code:openapi:update` updates an existing (or creates new if file doesn't exist) OpenAPI file with provided via CLI OpenAPI shcema converted to JSON.

Reviewer: ```suggestion
The `vendor/bin/syncapi code:openapi:update` updates an existing OpenAPI file (or creates a new one if the file doesn't exist) with the provided JSON.
```
